docker-compose.yaml

```
@@ -177,3 +177,15 @@ services:
     volumes:
       - ./.env:/opt/.env:ro
       - ./contracts.json:/opt/contracts.json:ro
+  dipper:
+    container_name: dipper
+    build: { context: dipper }
+    depends_on:
+      subgraph-deploy: { condition: service_completed_successfully }
+    ports:
+      - "${DIPPER_ADMIN_RPC_PORT}:${DIPPER_ADMIN_RPC_PORT}"
+      - "${DIPPER_INDEXER_RPC_PORT}:${DIPPER_INDEXER_RPC_PORT}"
```

This facilitates changing the ports used in .env without breaking the mapping to the container port.
```suggestion
      - "${DIPPER_ADMIN_RPC_PORT}:9000"
      - "${DIPPER_INDEXER_RPC_PORT}:9001"
```